client/homescreen/activity-panel/style.scss

```diff
@@ -1,30 +1,47 @@
+@mixin accordion-header {
+	.woocommerce-accordion-header {
+		order: 1;
+		text-align: left;
+		.woocommerce-accordion-title {
+			vertical-align: bottom;
+			margin-right: $gap;
+		}
+	}
+}
 .woocommerce-accordion-card {
+	.components-card__header,
+	.components-panel__body-title .components-button {
```

Comment:
This is overlapping css between the components button and the header.
css for the empty card-header.
Although maybe this css should be moved to the accordion component?

Reply:
It could make sense to,but I'll leave it up to you, if you wanna do that as a follow up later.

Reply:
Yeah I suppose the same goes for the activity card, we will probably have to consolidate the css once @jeffstieler stock migration is finished as well, given the Activity Card will largely be used on the home screen instead of the header. It looks like the Inbox would be the only one still using it.

Reply:
in case this factors in, just adding a note here that we're going to remove the Inbox from the app header altogether so it will only appear on the Home screen.